python_modules/libraries/dagster-aws/dagster_aws/ecs/container_context.py

```diff
@@ -189,7 +189,6 @@
                 ),
                 "replica_count": Field(
                     int,
-                    default_value=1,
```

Comment:
Why?

Reply:
if there's default config values like this, we don't have a way to tell between 'user set this' and 'user did not set this' when determining which config has been set